app/experimenter/normandy/tasks.py

```diff
@@ -38,7 +38,8 @@ def update_recipe_ids_to_experiments():
 
     ready_to_ship_experiments = Experiment.objects.filter(
         status__in=[Experiment.STATUS_SHIP, Experiment.STATUS_ACCEPTED]
-    )
+    ).exclude(type=Experiment.TYPE_RAPID)
```

Comment:
should rapid experiments be excluded [here](https://github.com/mozilla/experimenter/pull/3013/files#diff-0f18d819e50bfd722ece30241e99a6f9R77) too?

Reply:
That link doesnt' seem to work from me, which line were you referencing?

Reply:
gah, line 77 `update_launched_experiments`

Reply:
Oh good call!  I think it would do nothing since it wouldn't find anything in normandy for those slugs anyways, but it's still making unnecessary calls 👍